janusgraph-dist/src/assembly/static/conf/logback.xml

```diff
@@ -1,12 +0,0 @@
-<configuration>
```

Comment:
Seems `* src/assembly/static/cassandra/conf/logback.xml` from `janusgraph-dist/README.md` needs to be removed.

In addition, I wonder why we remove this file.

Reply:
No this is required for Cassandra distribution.

Reply:
Ya, my bad. Btw do we remove this file because we are not using `logback` anyways?

Reply:
We are replacing it.